sql-statements/sql-statement-import-into.md

```diff
@@ -36,6 +36,7 @@ For TiDB Self-Hosted, `IMPORT INTO` supports importing data from files stored in
 - When the [Global Sort](/tidb-global-sort.md) feature is used for data import, the data size of a single row after encoding must not exceed 32 MiB.
 - When the Global Sort feature is used for data import, if the target TiDB cluster is deleted before the import task is completed, temporary data used for global sorting might remain on Amazon S3. In this case, you need to delete the residual data manually to avoid increasing S3 storage costs.
 - Ensure that the data to be imported does not contain any records with primary key or non-null unique index conflicts. Otherwise, the conflicts can result in import task failures.
+- For the executed `IMPORT INTO` tasks based on scheduled by the Distributed eXecution Framework (DXF), if the task is already running, it is unsupported to be scheduled on a new TiDB nodes. If the TiDB nodes executing the import tasks is restart, the TiDB nodes will no longer execute the import task, but transfer to other TiDB nodes to continue executing. If import local TiDB nodes data, it will not failover to other TiDB nodes after the tasks exception.
```

Comment:
```suggestion
- If an `IMPORT INTO` task scheduled by the Distributed eXecution Framework (DXF) is already running, it is not supported to be scheduled to a new TiDB node. If the TiDB node that executes the data import task is restarted, it will no longer execute the data import task, but transfers the task to another TiDB node to continue executing. However, if the imported data is a local file, TiDB will not transfer the task to another TiDB node if the task fails.
```

Reply:
Replace “ TiDB will not transfer the task to another TiDB node if the task fails” with “The task will not be transfered to another TiDB node to continue executing.

Reply:
Thanks. fix it done.